wiki/Welcome/en.md

```diff
@@ -1,148 +1,37 @@
 
-You will have a few tabs like "\#osu!", "\#announce", "\#userlog" or if you're in a conversation with someone, you'll see their name as a separate tab. Navigating through the tabs is extremely easy, just press on the tab and type your message then press "ENTER" to send the message to the opened tab. Regardless, please do not spam (continuously sending short messages into the chat) as doing so will automatically mute your account for a few minutes and your "hard-work" removed from the chat. You can press "F9" to open the user list (called Extended Chat Console) connected to the server, and you can apply filter to narrow down your search.
+Modding is the process of users review (or commonly called "mod") a creator's beatmap in the pending (or work in progress/help) stage. Modding plays a big role in quality control because this allows creators to fix issues with their beatmap.
 
-![Chat Console friendlist][Chat_Console friendlist image]
+## osu!wiki - How you can help!
 
-Those are the basics, don't be shy to ask for help in-game or in forums, also the osu!wiki is here to help understand the game more.
+Since 26. August 2016, the osu!wiki has moved to a GitHub repository. After this change, the workflow has drastically changed. All contributions now go through the osu-wiki GitHub repository and are reviewed before being approved and merged to the osu!wiki.
 
-## Beatmapping (Edit)
-_Full explanation: [Beatmapping][Beatmapping wikilink]_
+To get started, see the [osu!wiki Contribution Guide](/wiki/owcg).
 
-osu! offers an opportunity to any player to create their own beatmaps. If you think you're creative and passionate about learning, you may start mapping your own songs. To start, simply access the "Edit" mode and drag your song's **.mp3** file onto the editor. Press "F5" to refresh the song list and you should find a blank map with your song name in it. However, beatmapping is a _very_ complicated process which requires a bit of dedication, and thus it will not be covered on this page. The best way to learn is to try it out yourself. You may read useful guides to get you started with mapping [right here][Beatmapping/Modding: Guide Compendium link]. Also, "Edit" mode can also be useful to take a closer look at any beatmap so you can see whats up with that part where you always fail! Also try to **avoid editing ranked maps** because they will automatically revert to being **unranked** (locally only of course) and you will not be able to earn pp/accuracy from it until you update the map again.
+## Common points of interest
```

Comment:
maybe this header should be about the Index page?

Reply:
See #94 
> Branch into links to pages for common points of interest, such as individual gamemodes and "Getting Started" style pages for each one

Why[?] the index page (is well out of date but) doesn't really tell people where the commonly wanted resources are located. (Well, if you say that the current links under it don't fulfill the common points of interest, just remember that this is a draft and that I have no clue what to put in it)